### ISSUE TITLE
Returns string cells instead of integer in some cases

### ISSUE THREAD

**Issue by wolfgang371** [d1ceward/xlsx-parser#3]
your sample.xlsx is built e.g. as follows:
```xml
<c r="A1" s="1" t="s"><v>0</v></c>
<c r="G2" s="0" t="n"><v>10</v></c>
```

if I write an .xlsx via `Monpt7/crexcel`, it looks e.g. like the following:
```xml
<c r="A1" t="s"><v>0</v></c>
<c r="A2">            <v>1</v></c>
```
so, for numbers the type seems to be omitted. If I open this file with Excel, it works fine - so maybe "n" is the default? I couldn't find it, though.
However, when read back in with xlsx-parser, those cells are shown as strings.

**Comment by d1ceward** [comment]
Thanks for the report, I just made a release with a fix (1.0.4), it should be good, it's a silly error where the default type was explicitly a string in the parsing step and prevent the automatic detection of the type afterwards.

```crystal
# src/xlsx-parser/sheet.cr:62
@book.style_types[cell_style_idx.try(&.to_i) || 0] # guilty code '|| 0' 
```

**Comment by wolfgang371** [comment]
thanks for the quick fix, works for me now!
